Bottom: cc360c942d6074f5332839aece27a20e1043f04d
Top:    5ca11089d9cb33c2034ff3c3ce30dc06da21e666
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 09:17:44 -0500

Add new dependency, bc, to Dockerfile


---
diff --git a/docker/buildbot-worker-ptxdist/Dockerfile b/docker/buildbot-worker-ptxdist/Dockerfile
index ee6bfeb5..b14d49c4 100644
--- a/docker/buildbot-worker-ptxdist/Dockerfile
+++ b/docker/buildbot-worker-ptxdist/Dockerfile
@@ -22,6 +22,7 @@ RUN apt-get update \
 &&  DEBIAN_FRONTEND=noninteractive \
     apt-get -y install -q \
         autoconf \
+        bc \
         bison \
         build-essential \
         ccache \
